File: Capitulo05-Exercicio05.c
// ADS 2020 - LINGUAGEM DE PROGRAMAÇÃO
// CAPÍTULO 05 - EXERCÍCIO 04
/* Criar em linguagem C utilizando o comando WHILE, um programa que imprima uma palavra infinitamente,
até que uma tecla seja pressionada*/

#include<stdio.h>
#include<stdlib.h>
#include<conio.h>
#include<locale.h>
#include<time.h>

main()
{
    setlocale(LC_ALL,""); // Configura a linguagem para o protugues do Brasil
    system("cls"); // Limpa a tela do terminal.
    while(!kbhit()) // Repete a rotina até que uma tecla seja precionada.
    {
        printf(" ... Repita ");
        sleep(250);
    }
    printf("\n\nFoi precionada uma tecla ...\nAté mais!!!");
    sleep(2000);
    
}

File: Capitulo05-Exercicio05.py
# ADS 2020 - LINGUAGEM DE PROGRAMAÇÃOO
# CAPÍTULO 05 - EXERCÍCIO 05
''' Criar em linguagem C utilizando o comando WHILE, um programa que
imprima uma palavra infinitamente, até que uma tecla seja pressionada'''

from pynput.keyboard import Listener, Key
from time import sleep

def press(key):
    global sair
    sair = True

with Listener(on_press=press) as keyb:
    keyb.join()
print("CAPÍTULO 05 - EXERCÍCIO 05")
while sair == False:
    print("Fatec, ")
    sleep(0.5)